task/06-conditions-n-loops-tasks.js

```diff
@@ -46,10 +53,15 @@ function getFizzBuzz(num) {
  *   10 => 3628800
  */
 function getFactorial(n) {
-    throw new Error('Not implemented');
+	//  throw new Error('Not implemented');
+	for (let x = 1, i = 1; i <= n; i++) {
+		x *= i; 
+		if(i == n) return x
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
""0"" == 0
and ""0"" === 0)"

Reply:
fixed